packages/cli/src/utils/generate.ts

```diff
@@ -371,11 +372,24 @@ function checkDependencies(basePath: string, packagesToCheck: string[]) {
         )} - Version mismatch detected for ${packageName}. 
           Core: ${coreVersion}, Customization: ${rootVersion}. Please align both versions to prevent issues`
       )
-     
     }
   })
 }
 
+function updateNextConfig(basePath: string) {
+  const { tmpDir } = withBasePath(basePath)
+
+  const nextConfigPath = path.join(tmpDir, 'next.config.js')
+
+  let nextConfigData = String(readFileSync(nextConfigPath))
+  nextConfigData = nextConfigData.replace(
+    /outputFileTracingRoot\:\s+(.*),/,
+    `outputFileTracingRoot: '${process.cwd()}',`
+  )
```

Comment:
If the `outputFileTracingRoot` isn't on the config, this will throw.

Reply:
Why would it? It would just return the original string, no? 

<img width="598" alt="image" src="https://github.com/user-attachments/assets/6a05cb1b-fe86-4279-be84-4bad785f3c78">